pkg/config/defaults.go

```diff
@@ -75,6 +75,7 @@ var (
 	defaultLogFormat                     = LogFormatText
 	defaultLogLevel                      = LogLevelInfo
 	defaultLogForward                    = false
+	defaultLoggingRetryLimit             = "1"
```

Comment:
[👋] It would probably make more sense to use a saner default value, like 3 or 5. (remember to also update the comment in `config.go` if you end up changing this value). This would provide a greater reliability to the Log Forwarder, transparently to the end user.

Reply:
I've set 1 because is the current value for our customers by default, do we want to change that?

Reply:
As discussed outside github, we'll go with 5 retry as default instead one.